src/Align.jl

```diff
@@ -0,0 +1,15 @@
+# Bio.Align
+# =========
+#
+# Module for biological sequence alignment.
+#
+# This file is a part of BioJulia.
+# License is MIT: https://github.com/BioJulia/Bio.jl/blob/master/LICENSE.md
+
+module Align
+
+using Reexport
+
+@reexport using BioAlignments
```

Comment:
Why not `const Align = BioAlignments`? I mean `Bio.Align` is an alias of `BioAlignments`.